internal/install/execution/terminal_reporter.go

```diff
@@ -65,20 +65,31 @@ func (r TerminalStatusReporter) ReportRecipeAvailable(status *StatusRollup, reci
 }
 
 func (r TerminalStatusReporter) ReportComplete(status *StatusRollup) error {
-
 	if status.hasFailed() {
 		return fmt.Errorf("one or more integrations failed to install, check the install log for more details: %s", status.LogFilePath)
 	}
 
-	msg := `
-  Success! Your data is available in New Relic.
+	recs := status.recommendations()
+
+	if len(recs) > 0 {
+		fmt.Println("  ---")
+		fmt.Println("  Instrumentation recommendations")
+		fmt.Println("  We discovered some additional instrumentation opportunities")
```

Comment:
nit: Maybe add a colon at the end of the message?

Reply:
👍🏼